docs/templates/index.rst.jinja2

```diff
@@ -85,6 +85,8 @@ you specify in ``ProfileConfig``.
 
 Disabling dbt event tracking
 --------------------------------
+
+    Note: deprecated and will be removed in 2.0.0. Use dbt_config_vars instead.
 .. versionadded:: 1.3
```

Comment:
I'd suggest moving the note into a sphinx note block like below and  you should also mention how to use dbt config vars to have the same effect:
```
.. note:
   Deprecated in v.1.4 and will be removed in v2.0.0. Use dbt config vars instead with ....
.. versionadded:: 1.3
```